scripts/auto_collect_brainglobe_tools_pub.py

```diff
@@ -0,0 +1,192 @@
+import pyalex
+import os
+from datetime import datetime
+
+BRAINGLOBE_CORE_WORKS = [
+    "https://openalex.org/W3138257762",
+    "https://openalex.org/W3007765542",
+    "https://openalex.org/W3165457844",
+    "https://openalex.org/W3092644694",
+]
```

Comment:
Just to make sure we get everything, there are a few different versions of these papers

```suggestion
BRAINGLOBE_CORE_WORKS = [
    "https://openalex.org/W3138257762",
    "https://openalex.org/W3007765542",
    "https://openalex.org/W3165457844",
    "https://openalex.org/W3092644694",
    "https://openalex.org/W4206584351"
    "https://openalex.org/W3209832784"
    "https://openalex.org/W3165052512"
    "https://openalex.org/W3205056304"
]
```

Reply:
Thanks